### PR TITLE
Scale and rotate swag images

### LINKED ISSUE #245: Make images interactive
- [x] I've checked that this isn't a duplicate issue.

When I saw the website the first thing I tried to do was a click on an image. How about we made the pictures interactive (and maybe clicking them served the purpose of a link)?

### PR DESCRIPTION
**Pull request by web-man**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Fixes #245 

Added some scale and rotate animations when user hovers item. If you don't like rotate just tell me I will remove it easily.

### CODE REVIEW

**Comment by Yash-Garg** [comment]
Would be nice if u add a gif here, so that everyone can see how it looks :)

**Comment by web-man** [comment]
@Yash-Garg Netlify gives us [link](https://deploy-preview-264--peaceful-chandrasekhar-efde8e.netlify.com/) to check changes.

**Comment by Yash-Garg** [comment]
OwO

**Comment by aslafy-z** [comment]
I personaly like it but I think there should be some more random in it. The images are all rotating in the same direction with the same angle. Thoughts?

**Comment by plibither8** [comment]
On mobile, the animation happens when the user _clicks/taps_  on the image rather than _hover_, since that's not possible. The image stays rotated and enlarged until another image is tapped. I think there should be an easier way to "reset" the image to its originals configs, perhaps when tapping/clicking the image again?

Otherwise, I like it :)

File: src/styl/imports/card.styl [hunk 1]
```diff
@@ -56,9 +56,16 @@
...
 
 	.img-container
 		height 300px
 		width 100%
 		overflow hidden
 		padding-bottom 15px
+
+	&:hover
+		img
+			-ms-transform rotate(3deg) scale(1.05)
```
Comment (vikaspotluri123) [inline src/styl/imports/card.styl]: @aslafy-z do we have autoprefixer as part of our build system?

Comment (aslafy-z) [reply src/styl/imports/card.styl]: It seems not, but it could be great to add it.

Comment (vikaspotluri123) [reply src/styl/imports/card.styl]: Created an issue for it 👍

File: src/styl/imports/card.styl [hunk 2]
```diff
@@ -56,9 +56,16 @@
 	img
 		height 100%
 		width auto
+		transition: all 300ms linear;
```
Comment (aslafy-z) [inline src/styl/imports/card.styl]:
```suggestion
		transition: all 300ms linear
```

File: src/styl/imports/card.styl [hunk 3]
```diff
@@ -56,9 +56,16 @@
...
 		height 300px
 		width 100%
 		overflow hidden
 		padding-bottom 15px
+
+	&:hover
+		img
+			-ms-transform rotate(3deg) scale(1.05)
+			-webkit-transform rotate(3deg) scale(1.05)
+			transform rotate(3deg) scale(1.05)
```
Comment (aslafy-z) [inline src/styl/imports/card.styl]: Since we merged #301, you can drop the two vendor specific lines (-ms-transform & -webkit-transform).

**Comment by swapagarwal** [comment]
Looks good to me, can be merged after resolving the conflicts.

**Comment by web-man** [comment]
@swapagarwal Done

**Comment by swapagarwal** [comment]
Thanks @web-man! 👍

**Comment by web-man** [comment]
@swapagarwal You're welcome!

**Pull request merged**